app/main.py

```diff
@@ -1,56 +1,49 @@
-#from typing import List
-#from uuid import UUID
-from fastapi import FastAPI, HTTPException
+from typing import List
+# from uuid import UUID
+import uvicorn
+from fastapi import FastAPI, HTTPException, Depends
+from pydantic import BaseModel
 from pymongo import MongoClient
-
-#from models.models import Gender, Role, User
+from pymongo.database import Database
 
 app = FastAPI()
 
-client = MongoClient(host="db")
-db = client["government_catnip"]
-
-#db: List[User] = [
-#    User(
-#        id="31b6e462-4c9a-4d3d-a00a-e7a8fc43c4e7", 
-#        first_name="Jamila", 
-#        last_name="Ahmed",
-#        gender=Gender.female,
-#        roles=[Role.student]
-#    ),
-#    User(
-#        id="03dfb08b-faa6-47fc-b60c-3484a62f8b2a", 
-#        first_name="Alex", 
-#        last_name="Jones",
-#        gender=Gender.male,
-#        roles=[Role.admin, Role.user]
-#    )
-#]
+
+async def get_db():
+    client = await check_connect_mongodb()
+    db = client["government_catnip"]
+    return db
+
+
+async def check_connect_mongodb():
+    try:
+        client = MongoClient(host="db")
+        client.server_info()
+    except Exception:
+        raise HTTPException(status_code=500, detail=f"Unable to connect to the server")
+    return client
+
 
 @app.get("/")
 async def root():
     return {
         "message": "Sup for our documentation go to link variable",
         "link": "https://catnip-govenment-module.github.io/government-catnip"
     }
-    
-#@app.get("/api/v1/users")
-#async def fetch_users():
-#    return db;
-#
-#@app.post("/api/v1/users")
-#async def register_user(user: User):
-#    db.append(user)
-#    return {"id": user.id}
-#
-#@app.delete("/api/v1/users/{user_id}")
-#async def delete_user(user_id: UUID):
-#        for user in db:
-#            if user.id == user_id:
-#                db.remove(user)
-#                return
-#        raise HTTPException(
-#            status_code=404,
-#            detail=f"user with id: {user_id} does not exists"
-#        )
-#
\ No newline at end of file
+
+
+class Location(BaseModel):
+    location_id: int
+    location: str
+    population: int
+    numberOfVoters: int

```

Comment:
Can we move the model to another directory like ```app/models/``` so that the code appears cleaner